Source/Windows/Common/InvalidationTracker.cpp

```diff
@@ -177,4 +177,18 @@ bool InvalidationTracker::HandleRWXAccessViolation(uint64_t FaultAddress) {
   }
   return false;
 }
+
+FEXCore::HLE::ExecutableRangeInfo InvalidationTracker::QueryExecutableRange(uint64_t Address) {
+  std::shared_lock Lock(IntervalsLock);
+  const auto XResult = XIntervals.Query(Address);
+  if (!XResult.Enclosed) {
+    return {};
+  }
+  const auto RWXResult = RWXIntervals.Query(Address);
+  if (RWXResult.Enclosed || (RWXResult.Size && RWXResult.Size < XResult.Size)) {
+    return {RWXResult.Size, RWXResult.Enclosed};
+  }
+  return {XResult.Size, false};
```

Comment:
Am I misreading this or is the size returned between the Linux SMC tracking and this tracking different?
It looks like this WIN32 implementation returns the full size of the region that contains the address, while the Linux SMC side is returning the size of the region minus the offset of the address queried? So full size on Win32 versus Remaining size on Linux?

Reply:
Query.Size is the size from the address queried, not the size of the region

Reply:
Alright cool, that clears up the confusion.